### ISSUE TITLE
Difference in bnml-restructure branch from master one

### ISSUE THREAD

**Issue by ilizol** [t-bullock/kassia#12]
`master` branch:

```xml
        <neumes>
            bare apos_apos
        </neumes>
        <neumes color="#cf232b">
            gorgUR
        </neumes>
        <neumes>
            olig
        </neumes>
        <lyrics>
            set _
        </lyrics>
```
![master](https://user-images.githubusercontent.com/7188298/107427095-b2a63400-6b29-11eb-8f1a-e18b785dbf4d.png)

`bnml-restructure` branch:

```xml
            <syllable>
                <lyric>set</lyric>
                <neume-group>
                    <neume>bare</neume>
                    <neume type="primary">apos</neume>
                </neume-group>
            </syllable>
            <syllable>
                <lyric>_</lyric>
                <neume-group>
                    <neume>apos</neume>
                    <neume type="chronos">gorgUR</neume>
                </neume-group>
            </syllable>
            <syllable>
                <lyric>_</lyric>
                <neume-group>
                    <neume>olig</neume>
                </neume-group>
            </syllable>
```
![bnml-restructure](https://user-images.githubusercontent.com/7188298/107427102-b5088e00-6b29-11eb-894b-334dcf114195.png)

**Comment by ilizol** [comment]
`master` branch:
```xml
        <neumes>
            peta_isonU_klasD bare apos_apos
        </neumes>
        <lyrics>
            un- to
        </lyrics>
```
![master](https://user-images.githubusercontent.com/7188298/107428487-60661280-6b2b-11eb-843c-f0ab289243c6.png)

`bnml-restructure` branch:
```xml
            <syllable>
                <lyric>un-</lyric>
                <neume-group>
                    <neume>peta</neume>
                    <neume>isonU</neume>
                    <neume>klasD</neume>
                </neume-group>
            </syllable>
            <syllable>
                <lyric>to</lyric>
                <neume-group>
                    <neume>bare</neume>
                    <neume type="primary">apos</neume>
                    <neume>apos</neume>
                </neume-group>
            </syllable>
```
![bnml-restructure](https://user-images.githubusercontent.com/7188298/107428494-61973f80-6b2b-11eb-944a-a6812c0dfa2f.png)

**Comment by ilizol** [comment]
`master` branch:
```xml
        <neumes>
            bare ison_apos
        </neumes>
        <neumes color="#cf232b">
            gorgUR
        </neumes>
        <neumes>
            olig
        </neumes>
        <lyrics>
            to _
        </lyrics>
```
![master](https://user-images.githubusercontent.com/7188298/107430374-bd62c800-6b2d-11eb-9281-90f7c027bdd5.png)

`bnml-restructure` branch:
```xml
            <syllable>
                <lyric>to</lyric>
                <neume-group>
                    <neume>bare</neume>
                    <neume type="primary">ison</neume>
                    <neume>apos</neume>
                    <neume type="chronos">gorgUR</neume>
                </neume-group>
            </syllable>
            <syllable>
                <lyric>_</lyric>
                <neume-group>
                    <neume>olig</neume>
                </neume-group>
            </syllable>
```
![bnml-restructure](https://user-images.githubusercontent.com/7188298/107430715-34985c00-6b2e-11eb-8cda-b373d4925162.png)

**Comment by t-bullock** [comment]
Fixed in commit #13.

**Comment by ilizol** [comment]
Only lyrics position fixed with commit #13. There is also the stacking issue.
